jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/helper/BuildUtil.java

```diff
@@ -13,22 +13,25 @@
  */
 package org.eclipse.jkube.kit.build.api.helper;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.Optional;
+
+import javax.annotation.Nonnull;
+
 import org.eclipse.jkube.kit.build.api.assembly.AssemblyManager;
 import org.eclipse.jkube.kit.common.JKubeConfiguration;
 import org.eclipse.jkube.kit.common.JavaProject;
 import org.eclipse.jkube.kit.config.image.build.BuildConfiguration;
 
-import javax.annotation.Nonnull;
```

Comment:
This class was completely reformatted and is really hard to track the changes (at least in the GH interface).
Could you please revert the reformatting so that it's clearer for us to see what was changed here?